src/ncp/posix/netif.cpp

```diff
@@ -54,13 +56,55 @@ otbrError Netif::Dependencies::Ip6Send(const uint8_t *aData, uint16_t aLength)
 {
     OTBR_UNUSED_VARIABLE(aData);
     OTBR_UNUSED_VARIABLE(aLength);
+
     return OTBR_ERROR_NONE;
 }
 
+otbrError Netif::Dependencies::Ip6MulAddrUpdateSubscription(const otIp6Address &aAddress, bool aIsAdd)
+{
+    OTBR_UNUSED_VARIABLE(aAddress);
+    OTBR_UNUSED_VARIABLE(aIsAdd);
+
+    return OTBR_ERROR_NONE;
+}
+
+#if OTBR_NETIF_USE_MLD_MONITOR
```

Comment:
Is it better to use MLD only?

Reply:
This isn't a configuration but an evaluation if we should use MLD monitor. In OT posix netif.cpp:
```
#if defined(RTM_NEWMADDR) || defined(__NetBSD__)                                                                                                                                                                                                       
// on some BSDs (mac OS, FreeBSD), we get RTM_NEWMADDR/RTM_DELMADDR messages, so we don't need to monitor using MLD                                                                                                                                    
// on NetBSD, MLD monitoring simply doesn't work                                                                                                                                                                                                       
#define OPENTHREAD_POSIX_USE_MLD_MONITOR 0                                                                                                                                                                                                             
#else                                                                                                                                                                                                                                                  
// on some platforms (Linux, but others might be made to work), we do not get information about multicast                                                                                                                                              
// group joining via AF_NETLINK or AF_ROUTE sockets.  on those platform, we must listen for IPv6 ICMP                                                                                                                                                  
// MLDv2 messages to know when multicast memberships change                                                                                                                                                                                            
//              https://stackoverflow.com/questions/37346289/using-netlink-is-it-possible-to-listen-whenever-multicast-group-membership-is-ch                                                                                                          
#define OPENTHREAD_POSIX_USE_MLD_MONITOR 1                                                                                                                                                                                                             
#endif // defined(RTM_NEWMADDR) || defined(__NetBSD__)       
```
It said when `defined(RTM_NEWMADDR) || defined(__NetBSD__)` MLD doesn't work. So I think we can use macro to turn this off on these platforms. Thoughts?

Reply:
As discussed offline, we will always use MLD_MONITOR. Will do special handling on some platforms where MLD_MONITOR doesn't work.

Reply:
Updated!